django_api/Dockerfile

```diff
@@ -15,6 +15,6 @@ RUN useradd nonroot && chown nonroot: /api
 USER nonroot
 
 # Specify the entrypoint script
-ENTRYPOINT ["./entrypoint.sh"]
```

Comment:
dont modify this file, plaese. We have [PR](https://github.com/Iron-Cow/MonoProject/pull/9) with alternative ways of launch application

Reply:
Done